src/app/public/modules/format/app-format.spec.ts

```diff
@@ -0,0 +1,18 @@
+// #region imports
```

Comment:
@Blackbaud-SteveBrush While I love collapsible regions, I noticed that VS Code will collapse imports automatically. Is this just being thorough?

Reply:
Huh, that must be a recent feature to VSCode. We should remove this requirement from the style guide, too.